experimental/saiextensions.h

```diff
@@ -81,6 +82,8 @@ typedef enum _sai_api_extensions_t
 
     SAI_API_DASH_HA,
 
+    SAI_API_DASH_ROUTING_GROUP,
```

Comment:
why this is not added at the end ? such changes will not be backward compatible later

Reply:
@r12f is there a way to control ordering in this enum?

Reply:
The current logic of generation is that we will scan all the existing attributes and ignore if the attribute is already added.

Since you are hitting this issue, it should be caused by you have generated the SAI headers in multiple branches without resetting. If you revert all the SAI changes by `git reset --hard` and regenerate it, it will be added in the end.

Reply:
There's no longer a separate API, this change is removed